WooCommerce/Classes/ViewRelated/Dashboard/Stats v4/StoreStatsPeriodViewModel.swift

```diff
@@ -204,16 +204,9 @@ private extension StoreStatsPeriodViewModel {
                 return nil
             }
         }
-        guard let selectedIndex = selectedIntervalIndex else {
-            return StatsTimeRangeBarViewModel(startDate: startDate,
-                                              endDate: endDate,
-                                              timeRange: timeRange,
-                                              timezone: siteTimezone)
-        }
-        let date = orderStatsIntervals[selectedIndex].dateStart(timeZone: siteTimezone)
+
         return StatsTimeRangeBarViewModel(startDate: startDate,
                                           endDate: endDate,
-                                          selectedDate: date,
```

Comment:
Let's not alter this logic as we still want to display the selected time/date when selecting a range on the chart.